src/main/java/frc/robot/FieldConstants.java

```diff
@@ -59,6 +59,8 @@ public static boolean isRedAlliance() {
         return alliance.isPresent() && alliance.get() == Alliance.Red;
     }
 
+    public static final Translation2d SPEAKER = new Translation2d(0, 5.5461679526); //Pose should be correct
```

Comment:
Really. I think that goes to nanometers!!! What is the smallest distance we care about on the field?

Please trim to 2 decimal places.